webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
```

Comment:
I think webpack is too much for this, why not using PostCSS along with Rollup? Is faster and lightweight.